.scalafmt.conf

```diff
@@ -3,4 +3,5 @@ assumeStandardLibraryStripMargin = true
 project.excludeFilters = [
   "main/g8"
   "output/src"
+  "scalafix"
```

Comment:
why?

Reply:
nevermind, got it - the output is not properly formatted

Reply:
Exactly!